### ISSUE TITLE
[Tracking Issue] Zoom passcode update

### ISSUE THREAD

**Issue by mrbobbytables** [kubernetes/community#5146]
We have some changes that need to be made to everyone's zoom meetings. :(

TL;DR

- Zoom will require passcode or waiting room enabled by September 27th
- Passcode was chosen as our current practices are good enough and waiting room isn’t practical
- Set the passcode to your meetings to be (five sevens): **77777**
- Update sigs.yaml, notes, and calendar invites with a new zoom link once passcode is set.

[On September 27th, Zoom will force all of our meetings to use a waiting room OR a passcode for people to enter](https://support.zoom.us/hc/en-us/articles/360045009111-FAQ-Meetings-Waiting-Room-and-Passcode-Requirements-September-27-2020). The bad news is we will all need to update our meetings. It’s something that each SIG must do on their own, as their SIG account.

After much discussion, we’ve come to the conclusion that setting a passcode for all our meetings is better than making each SIG lead have to login and manage a waiting room. Our current moderation practices have let us hold meetings in the open with minimal risk to zoom bombings or trolls. The passcode should pose little to no change from the current practices outside of updating the links and docs.

To make sure we’re consistent across the project we’ve decided to set one common passcode (Five Sevens): **77777**

We’ve bikeshedded 77777 as the passcode to set for all K8s meetings. Please add this information to your SIG Meeting notes. calendar entries, and sigs.yaml so that when new members want to join it’s obvious that they need the common passcode.

If you do NOT set a passcode, then starting from the 27th, your meeting will have the waiting room enabled. This means you must first login as the zoom account to start the meeting and will then have to manually allow people in during the call. This is incredibly annoying to do while running a meeting so we recommend setting the passcode before then.

Here are the instructions for what you need to do:
- Login to https://zoom.us with your `kubernetes-sig|wg|ug-foo-leads@googlegroups.com` account
- Goto Meetings
- For each meeting, press Edit, and then specify Edit All Occurrences
- Scroll down to security, and select the Passcode checkbox
- Set the passcode to: 77777
- Save the meeting.
- It will return you to the updated meeting event, copy the meeting invite link and update your calendar entries, docs, and sigs.yaml with the new link.

For those wondering what this means for our security posture, the current model of host/co-host having admin powers has been working well for us over the past two years, that part doesn’t change. However we felt that having SIGs assigning their own passcode would be error prone and increase the difficulty of joining, so we chose the lesser evil of **77777**.

---

**Checklist:**
- [x] steering committee
- [ ] sig-api-machinery
- [x] sig-apps
- [ ] sig-architecture
- [ ] sig-auth
- [ ] sig-autoscaling
- [x] sig-cli
- [x] sig-cloud-provider
- [x] sig-cluster-lifecycle
- [x] sig-contributor-experience
- [x] sig-docs
- [x] sig-instrumentation
- [ ] sig-multicluster
- [ ] sig-network
- [ ] sig-node
- [ ] sig-release
- [x] sig-scalability
- [ ] sig-scheduling
- [x] sig-security
- [ ] sig-service-catalog
- [ ] sig-storage
- [x] sig-testing
- [ ] sig-ui
- [ ] sig-usability
- [x] sig-windows
- [ ] ug-big-data
- [ ] ug-vmware-users
- [ ] wg-api-expression
- [ ] wg-component-standard
- [ ] wg-data-protection
- [x] wg-iot-edge
- [x] wg-k8s-infra
- [ ] wg-lts (maybe?)
- [ ] wg-multitenancy
- [x] wg-naming
- [ ] wg-policy
---
/sig contributor-experience
/area community-management
/milestone 1.20

**Comment by sftim** [comment]
[How] can we update the settings for existing Zoom meetings / will we need to create a new meeting and share updated links?

**Comment by mrbobbytables** [comment]
@sftim an email was sent to the leads mailing list, I have updated the tracking issue with the instructions 👍

**Comment by castrojo** [comment]
WG LTS reported that they might conclude and close down soon so I wouldn't sweat that one for now.

**Comment by jimangel** [comment]
SIG Docs is complete. I noticed that we link to our agenda which links to our zoom URLs so there is no need to update the community repo.

@sftim I updated it using the SIG Docs Zoom login (happy to share), then as a calendar owner, I was able to update the links. The way we had APAC setup, I was able to just add a passcode, for the regular weekly meeting, I had to create a new reoccurring one which enables us to alter settings in the future.

/cc @kbarnard10 @irvifa

**Comment by mrbobbytables** [comment]
I'm going to go ahead and close this out - with the comms regarding the updates, things should be okay and sigs.yaml will eventually be updated as people change meetings.

/close
